jobs/fxci-taskcluster-export/fxci_etl/schemas.py

```diff
@@ -92,12 +93,29 @@ class Tasks(Record):
     task_group_id: BigQueryTypes.STRING
     task_id: BigQueryTypes.STRING
     task_queue_id: BigQueryTypes.STRING
-    tags: list[Tag]
+    # No longer used. Kept for backwards compatibility with older records.
+    tags: Optional[list[Tag]]
 
     def __str__(self):
         return self.task_id
 
 
+@dataclass
+class Tags(Record):
+    task_id: BigQueryTypes.STRING
```

Comment:
It's unclear to me if this is new or not, but I'm curious why `task_id` is a tag. What sort of query would you write where it would be helpful to filter on this, or otherwise use it? I guess maybe it helps to group multiple runs from the same task together?

Reply:
Good question, each dataclass represents a different table in BigQuery, so the task_id is just needed to join it with the correct record in the `Tasks` table. I did consider putting all the tags into directly into the `Tasks` table (so there wouldn't be a separate `Tags` table.

Hmm, initially I didn't want to do that because I didn't want to pollute the `Tasks` table with a bunch of arbitrary tags.. but I guess they could still go in a nested column? I think I need to play around with this a bit first

Reply:
I'm fine with the current approach, to be clear, especially now that I understand that these are just different tables. This seems like classic 1:M relationship management in a database?

Reply:
Yeah, there's certainly nothing wrong with this approach.. But if I could leave this as a nested column, but still with the nested keys being well defined, I think that would make it simpler to query and would make the logic here a bit cleaner.

I think I want to at least test it out first before deciding which one to go with..

Reply:
Thanks for this comment, I think the latest push will make things much nicer! I'm not sure if you want to re-review or not.